responsibleai/responsibleai/_tools/causal/dashboard_schemas/schema_0.1.0.json

```diff
@@ -0,0 +1,220 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "AzureML Responsible AI causal analysis dashboard schema",
```

Comment:
probably should remove "AzureML Responsible AI" since this is OSS and not tied to AzureML in any way

Reply:
Oh definitely!! Thanks for catching this